src/servers/grpc_server_v2.cc

```diff
@@ -1209,8 +1231,63 @@ InferResponseAlloc(
         *actual_memory_type_id = 0;
       }
 
-      raw_output->resize(byte_size);
-      *buffer = static_cast<void*>(&((*raw_output)[0]));
+      if (data_format == DataFormat::EXPLICIT) {
```

Comment:
Check the updated V2 spec. There is no "data_format" anymore (I'm also updating our grpc_version_v2.proto appropriately as part of the shared-memory updates). So on the response side we should just always use the "raw_contents" field to return the response.